$NetBSD: patch-ad,v 1.1 2002/01/07 17:26:08 wennmach Exp $

"make clean" first, we don't really know what's left over from a
previous build in $BSDSRCDIR. Also, don't "make -j 4" (improves
log file legibillity for debugging purposes).

--- src/Makefile.kernel.orig	Fri Jan  4 10:48:02 2002
+++ src/Makefile.kernel	Fri Jan  4 10:51:18 2002
@@ -29,8 +29,9 @@
 	cp -p ${_KERNEL_CONF} ${.WORKDIR}/CONFIG
 	(cd ${.WORKDIR}; \
 		config -s ${_SYS_DIR} ;\
+		make clean;\
 		make depend;\
-		make -j 4 \
+		make \
 	)
 
 
